contracts/Aavegotchi/facets/ItemsFacet.sol

```diff
@@ -228,6 +228,8 @@ contract ItemsFacet is Modifiers {
         Aavegotchi storage aavegotchi = s.aavegotchis[_tokenId];
         require(aavegotchi.status == LibAavegotchi.STATUS_AAVEGOTCHI, "LibAavegotchi: Only valid for AG");
 
+        GotchiEquippedItemsInfo storage _equippedItemsInfo = s.gotchiEquippedItemsInfo[_tokenId];
```

Comment:
Since the pointer `_equippedItemsInfo` is only being passed as parameter. I suggest passing `tokenId` as parameter and reading from storage inside the functions